pkgs/by-name/cs/csvtk/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  ...
+}:
+let
+  version = "0.30.0";
+in
+buildGoModule {
+  pname = "csvtk";
+  inherit version;
```

Comment:
```suggestion
buildGoModule rec {
  pname = "csvtk";
  version = "0.30.0";
```

Reply:
Not sure about this one, though, but it seems to me the version using `rec` is more common.

Reply:
I tought that we are ditching the use of `rec` in favour of the `let in` pattern

Reply:
Oh, OK, I wasn't aware. So many new rules.